### DISCUSSION TITLE
Are there any plans to implement support for a static contact plugin?

### DISCUSSION THREAD

**Discussion by CryssDOTnet** [datenstrom/community#135] [Ask a question] [answered]
Are there any plans to implement support for a static contact plugin?

**Comment by markseuffert** [comment]
To be honest, I don't know how this would possible. Are there any services you're thinking of?

**Comment by CryssDOTnet** [comment]
I just don't want to use php online. I create the content offline and upload it. Maybe the easiest way would be to put some html forms in my index.html/index page.txt

**Comment by markseuffert** [comment, marked as answer]
I understand what you mean. That's the disadvantage of static websites, that they can't offer dynamic features like a contact page or online search. You would need some kind of web service, as far as I know. Here's what you can do: 
1. Install Yellow only for sending email and keep your content static 
2. Use as service like [Formspree](http://formspree.io) or [Simple Form](https://getsimpleform.com)

Hope this helps. :smile:

**Comment by CryssDOTnet** [comment]
Yes. Thank you!
